instance/private_parents.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright © LiquidWeb
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package instance
+
+import (
+	"fmt"
+	"strings"
+
+	apiTypes "github.com/liquidweb/liquidweb-cli/types/api"
+)
+
+// it helped package scope and getting access to the global client variable to
+// put functions within this package... I'm not sure if a bigger package is
+// more appropos that Client is a member of, or if this is actually fine..
+// or maybe the "instance" package name just makes it feel wrong...
+// feedback is welcome
```

Comment:
the original thought here as far as "instance" goes.. is because its "creating" an instance of lwCli, that we bring up for global use in `cmd/root.go`.

If you want to put command helpers like `DerivePrivateParentUniqId` in its own package, or commands themselves, like the `CloudServerCreate` you added to `instance`, I am fine with that. It might help keep things more organized in the future as more commands migrate to support plans.

If you do that, we should probably just make the new packages accessible under `instance.Client`. That way they are still easily accessible, via things like `ci.Cmds.CloudServerCreate` or `ci.CmdHelpers.DerivePrivateParentUniqId` etc

Reply:
that's a tough call, but the global variable is what makes it most difficult atm, and I didn't want to create a new one if I could avoid it...

I have an idea but my head hurts too much to fully articulate it so I'll not try right now, and it's not something that would prevent merging now because it doesn't change end functionality